chat2issue/cli.py

```
@@ -0,0 +1,598 @@
+
+def analyze_entries(args: argparse.Namespace) -> None:
+    ensure_directories()
+    entries = collect_inbox_entries()
+    if not entries:
+        raise Chat2IssueError("No inbox entries found. Use `ingest` first.")
+
+    if args.date:
+        try:
+            target_date = dt.datetime.strptime(args.date, "%Y-%m-%d").date()
+        except ValueError as exc:
+            raise Chat2IssueError(f"Invalid --date value: {args.date}") from exc
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)